Header header

float64 throttle
float64 footPedal
uint32 mode